ads/fakead3p.js

```diff
@@ -24,4 +24,23 @@ export function fakead3p(global, data) {
   if (data.ad_container) {
     dev().assert(data.container == data.ad_container, 'wrong container');
   }
+  if (data.valid && data.valid == 'true') {
+    const img = document.createElement('img');
+    if (data.url) {
+      img.setAttribute('src', data.url);
+    }
+    let width, height;
+    if (data.iframeheight) {
+      img.setAttribute('height', data.iframeheight);
+      height = Number(data.iframeheight);
+    }
+    if (data.iframewidth) {
+      img.setAttribute('width', data.iframewidth);
+      width = Number(data.iframewidth);
+    }
+    document.body.appendChild(img);
+    context.renderStart({width, height});
```

Comment:
I think one of adHeight/adWidth can be `undefined` and is still a valid resize request. (Just like ad resizing)
I should probably change the code to call `context.renderStart()` when both are `undefined`.